UI/window-basic-main.cpp

```diff
@@ -1083,7 +1083,9 @@ bool OBSBasic::InitBasicConfigDefaults()
 	config_set_default_uint  (basicConfig, "Audio", "SampleRate", 44100);
 	config_set_default_string(basicConfig, "Audio", "ChannelSetup",
 			"Stereo");
-
+	config_set_default_bool(basicConfig, "Audio", "enableMultichannel",
```

Comment:
I hadn't noticed before, but it seems that Jim _does_ have some commits in where he aligns the opening parenthesis after a function call.  Feel free to do the same, or leave them as is for Jim to comment on.